rotkehlchen/api/v1/schemas.py

```diff
@@ -1986,9 +1987,24 @@ def _transform_evm_addresses(data: dict[str, Any], ethereum_inquirer: 'EthereumI
         )
 
 
-class EvmAccountsPutSchema(AsyncQueryArgumentSchema):
+class ChainTypeSchema(Schema):
+    chain_type = SerializableEnumField(enum_class=ChainType, required=True)
```

Comment:
add a `limit_to` or an `exclude` argument just like we have for various other enums such as `LocationField`. This way we can  get rid of the extra 1 liner validation function `validate_chain_type` below